test/integration/framework.go

```diff
@@ -127,6 +139,7 @@ func withConfigGetFreshApiserverAndClient(
 	if nil != err {
 		t.Fatal("can't make the client from the config", err)
 	}
+	t.Logf("Test client will use API Server URL of %v", secureAddr)
```

Comment:
I feel like this gets printed in waitForAPIserverUp, but I suppose it's fine to print it as the final exit message.